server/task/task-json/src/main/java/org/apache/james/server/task/json/JsonTaskAdditionalInformationSerializer.java

```diff
@@ -49,7 +49,8 @@ public UnknownAdditionalInformationException(JsonGenericSerializer.UnknownTypeEx
 
     @Inject
     public JsonTaskAdditionalInformationSerializer(Set<AdditionalInformationDTOModule<?, ?>> modules) {
-        jsonGenericSerializer = new JsonGenericSerializer(modules);
+        //FIXME
```

Comment:
To be honest, I'm not able to keep track of all the failures each commit introduces because I'm not able to run the whole testsuite for each. For sure, it breaks things.